app/assets/stylesheets/menu.scss

```diff
@@ -36,4 +36,8 @@ label {
 .card-input-element:checked + .card-input {
   box-shadow: 0 0 1px 1px #2eaccc;
 }
-
+.hr{
```

Comment:
1. .hr { with space
2. If you want to set this style as generic, move it to some generic file, in other case add some specific behaviour